<launch>
  <arg name="raw_points_topic" default="points_raw" />
  <arg name="node_name" default="voxel_grid_filter" />
  <arg name="output_log" default="false" />

  <node pkg="points_downsampler" name="$(arg node_name)" type="$(arg node_name)">
    <param name="points_topic" value="$(arg raw_points_topic)" />
    <param name="output_log" value="$(arg output_log)" />

    <!-- Topic rename (IN) -->
    <remap from="config/voxel_grid_filter" to="config/voxel_grid_filter" />

    <!-- Topic rename (OUT) -->
    <remap from="/filtered_points" to="/filtered_points" />
    <remap from="/points_downsampler_info" to="/points_downsampler_info" />

  </node>
</launch>